helm/csi-isilon/templates/node.yaml

```diff
@@ -161,6 +167,37 @@ spec:
               mountPath: /registration
             - name: driver-path
               mountPath: /csi
+        {{- if hasKey .Values "authorization" }}
+        {{- if eq .Values.authorization.enabled true }}
+        - name: karavi-authorization-proxy
+          imagePullPolicy: Always
+          image: {{ required "Must provide the authorization sidecar container image." .Values.authorization.sidecarProxyImage }}
+          env:
+            - name: PROXY_HOST
+              value: "{{ .Values.authorization.proxyHost }}"
+            - name: INSECURE
+              value: "{{ .Values.authorization.insecure }}"
```

Comment:
We already have a param skipCertificateValidation in values.yaml for the same purpose. Can the same param be used here as well?

Reply:
This setting configures certificate validation for communication between the sidecar-proxy and proxy-server; not to backend storage. I will get back to you on this.

Reply:
Other drivers, at least csi-powerflex, do not have the skipCertificateValidation field in the values. We'd like to be as consistent as possible with how authorization is deployed so it makes sense to keep the additional field for authorization at this time.

Reply:
ok, agree with you on the additional field. But can we rename this env var/values field to skipCertificateValidation, as for similar requirements we use skipCertificateValidation var for driver? Please think about this.

Reply:
This change would require a code change in authorization since we read the INSECURE env variable [here](https://github.com/dell/karavi-authorization/blob/4549b8a9ef316905f910ff0db4c5892b63c06a27/cmd/sidecar-proxy/main.go#L212). We'd rather not have authorization code changes at this time but we have an item to address this in the future.